docs/en/observability/cloud-monitoring/aws/monitor-amazon-intro.asciidoc

```diff
@@ -18,20 +20,33 @@ deep insights into the performance of your applications, services, and
 infrastructure components.
```

Comment:
I think we can improve this intro paragraph a lot.

* Let's get rid of the `//TODO` on L6/7
* Remove the Kibana book-scoped attribute on L13/14
* L16-L20 uses a lot of words to not really say anything useful. How do we provide deep insights into the performance of apps, services, and infra? What can users monitor? Why should users be interested in monitoring AWS?

Reply:
Reworked the paragraph.